fastlane/spec/fixtures/xcodeproj/automatic_code_signing.xcodeproj/project.pbxproj

```diff
@@ -392,48 +392,64 @@
 			isa = XCBuildConfiguration;
```

Comment:
accidental change of test fixtures?

Reply:
It was reverted. I thought @taquitos fixed it, actually. At some point I'll replace that with a mock....